buildSrc/build.gradle

```diff
@@ -25,3 +25,5 @@ sourceSets{
  * Source: https://github.com/gradle/gradle/issues/17236#issuecomment-923074298
  */
 tasks.getByName("jar").setProperty("duplicatesStrategy", DuplicatesStrategy.EXCLUDE);
+sourceCompatibility = JavaVersion.VERSION_19
+targetCompatibility = JavaVersion.VERSION_19
```

Comment:
I think, this change has nothing to do with the fix. Please revert that change (if possible)

Reply:
addressed